img-placeholders/alert-info.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this new image directory for? Why can we not use the images in `core/img`?

Reply:
If you're talking about the `stylesheet/image/` directory, that's for placeholder images that are required for the stylesheet to look good but are not required (or even recommended) for someone building on top of decanter.

For example, `hero-banner.png` is a necessary placeholder for the hero component in the styleguide, but no one building on top of decanter needs (or wants) that specific image.

Contrast that with `caret-down-black.svg`, which is a necessary part of the nav component, so people building on top of decanter must have that specific image as part of their build.

Reply:
I was referring to the `img-placeholders` directory. What is this one for?

`core/img` is for assets required by decanter and `kss/builder/decanter/kss-assests` should be for all assets that are just for the styleguide and not for the decanter package. KSS node moves all files in this directory in to the styleguide build. In the non-webpack build I had an extra copy step to move the Decanter core package in to styleguide instead of duplicating them in the kss-assets directory.

Reply:
Shea moved the images to the right place.